src/dsp_tools/utils/connection.py

```diff
@@ -28,7 +28,7 @@ def get(
     def put(
         self,
         route: str,
-        jsondata: str | None = None,
+        jsondata: dict[str, Any] | None = None,
```

Comment:
could we rename `jsondata`/`jsonobj`, etc. to `data` everywhere? (also on the caller side)

Reply:
`data` would be fine for me, but I even prefer `payload`. What do you think?

Reply:
`data` would follow the naming used by requests. But I'm happy with either